File: MotivApp/app/src/main/java/com/motiv/piotr/BottomNavigationActivityContract.java
package com.motiv.piotr;


public interface BottomNavigationActivityContract {

    public interface View {

        public void navigationControllerstartDestinationById(int arg0);
    }

    public interface Presenter {

        public void eloonNavigationItemSelected(android.view.MenuItem argument0);
    }
}

File: MotivApp/app/src/main/java/com/motiv/piotr/BottomNavigationActivityContract.kt
 
package com.motiv.piotr

public interface BottomNavigationActivityContract {

    public interface View {

        public fun navigationControllerstartDestinationById(arg0: Int): Unit
    }

    public interface Presenter {

        public fun eloonNavigationItemSelected(argument0: android.view.MenuItem): Unit
    }
}
